### PR TITLE
Add polex for cloud-director's cloud provider app

### PR DESCRIPTION
**Pull request by jkremser**
### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/kyverno/templates/core-policies/cloud-director-cpi-polex.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{{- if .Values.policyExceptions.enableCloudDirectorProviderPolex }}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: cluster-service-exceptions
```
Comment (stone-z) [inline helm/kyverno/templates/core-policies/cloud-director-cpi-polex.yaml:5]: Name

File: helm/kyverno/values.yaml [hunk 2]
```diff
@@ -61,6 +61,9 @@ policyExceptions:
   # Deploy a PolicyException for chart-operator (required for Giant Swarm clusters).
   enableChartOperatorPolex: true
 
+  # Deploy a PolicyException for cloud-provider-cloud-director-app (required for Giant Swarm clusters).
+  enableCloudDirectorProviderPolex: false
```
Comment (stone-z) [inline helm/kyverno/values.yaml:65]: Same question here as the other PR -- I guess this is only CAPVCD? Is there a mechanism to enable it?

**Comment by fhielpos** [comment]
Is this still needed? Can we update/merge/close this?

**Comment by fhielpos** [comment]
@giantswarm/team-rocket , I will close this one. If you ever need this feel free to open it again.

**Pull request closed without merging**